packages/core/core/src/services/document-service/transform/relations/extract/data-ids.ts

```diff
@@ -57,7 +57,13 @@ const extractRelationIds = <T extends Schema.Attribute.RelationKind.Any>(
     // handle positional arguments
     const connect = Array.isArray(relation.connect) ? relation.connect : [relation.connect];
     connect.forEach((relation) => {
-      if (isShortHand(relation) || !('position' in relation)) return;
+      if (
+        isShortHand(relation) ||
+        relation === undefined ||
+        relation === null ||
+        !('position' in relation)
+      )
```

Comment:
```suggestion
      if (
      	!relation || 
        isShortHand(relation) ||
        !('position' in relation)
      )
```

Reply:
don't mind leaving it as it was, but just in case, is it intended?

Reply:
Thanks, I thought I'd done it